src/pages/createCostModelWizard/review.tsx

```diff
@@ -111,7 +112,14 @@ const ReviewDetailsBase: React.SFC<InjectedTranslateProps> = ({ t }) => (
                   {markup}%
                 </TextListItem>
                 <TextListItem component={TextListItemVariants.dt}>
-                  {t('cost_models_wizard.steps.sources')}
+                  {t('cost_models_wizard.steps.sources')}{' '}
+                  {sources.find(
+                    src => src.selected && src.costmodel !== undefined
+                  ) && (
+                    <WarningIcon
+                      text={t('cost_models_wizard.warning_override_sources')}
+                    />
+                  )}
```

Comment:
Wondering if the icon should appear on the right-most side? That is, instead of next to the label, located to the right of the value. Perhaps something to consider in a following PR?

<img width="794" alt="Screen Shot 2019-11-29 at 6 48 45 PM" src="https://user-images.githubusercontent.com/17481322/69892984-764a3780-12d9-11ea-8511-30287a5199d5.png">

Reply:
Yes, @nlcwong and I were taking about this and probably there is going to be a follow up.
Thanks.